python/Quickstart.MD

```diff
@@ -142,192 +119,201 @@ Body of event request:
-```
->>> comment_payload_file = 'comment_payload.json'
->>> comment_payload = utils.read_payload_file(comment_payload_file)
->>> comment_payload
-{'processId': '4ec1229a-d46f-4fc0-878f-188d318db55a', 'processType': 'directSupplierIncident', 'aptCommentBox': {'aptComment': {'commentText': 'Please provide a list of missing items and we will replace them.', 'visibilityType': 'Public'}}}
-```
-Your payload has been loaded properly, but the `processId` is still wrong. Use the `update()` function to change the stored value:
-```
->>> comment_payload.update({"processId": "fd04dd70-c162-43ef-8118-539ca01de471"})
->>> comment_payload
-{'processId': 'fd04dd70-c162-43ef-8118-539ca01de471', 'processType': 'directSupplierIncident', 'aptCommentBox': {'aptComment': {'commentText': 'Please provide a list of missing items and we will replace them.', 'visibilityType': 'Public'}}}
-```
-You can use the same process to change the `eventName`:
+**Type or copy the following into your Python terminal**  
 ```
->>> event_data.update({"eventName": "agile-process-teams:add-comment-for-incident:v1"})
->>> event_data
-{'eventName': 'agile-process-teams:add-comment-for-incident:v1', 'ownerId': '2aaeed40-3912-427e-a94f-51fd5a730345', 'processNetworkId': '61077cb2-601b-4c27-b11f-5e3910589931'}
+comment_request.get('header').update({'ownerId': owner_id})
+comment_request.get('header').update({'processNetworkId': network_id})
+
+comment_request.get('payload').update({'processId': incident_id})
 ```
-The payload and event data should now be updated. You can verify this by resending the commands to build the request body and resend the request.  
+After making our desired modifications, we can convert our dictionary to a JSON string and send our request.  
+
+**Type or copy the following into your Python terminal**  
 ```
->>> request_body = utils.create_payload(event_data, payload_data)
->>> comment_result = utils.post_reqeust(request_headers, requst_body)
+comment_request_json = json.dumps(comment_request)  
+comment_result = requests.request("POST", url, headers=request_header, data=comment_request_json)
 ```
 The response:  
 ```
 {
     "header": {
         "headerVersion": 1,
         "eventName": "agile-process-teams:add-comment-for-incident-response:v1",
-        "ownerId": "94f94f37-2772-4b39-8041-9c2dcfcfff82",
+        "ownerId": "YOUR_OWNER_ID",
         "isErr": false,
         "errCode": "200_OK",
-        "licensePlate": "vXPsFz"
+        "licensePlate": "LICENSE_PLATE_ID"
     },
     "payload": {
-        "id": "a67d7261-33b1-4be6-9760-61ae8fe9eed3"
+        "id": "COMMENT_ID"
     }
 }
 ```
 ## Read the details of your newly created incident
 In a previous step, you stored your incident id as a variable in your session.  You can use this stored variable to retrieve details about the incident.  
```

Comment:
Use the previously stored incident id to retrieve details about the incident.

Reply:
updated